enterprise/tasks.py

```diff
@@ -320,11 +322,13 @@ def send_group_membership_removal_notification(enterprise_customer_uuid, members
         [pecu_emails],
         ENTERPRISE_BRAZE_ALIAS_LABEL,
     )
-    for ecu in ecus:
-        recipients.append(braze_client_instance.create_recipient(
-            user_email=ecu['user_email'],
-            lms_user_id=ecu['user_id']
-        ))
+    for user_id_by_email_chunk in batch_dict(user_id_by_email, MAX_NUM_IDENTIFY_USERS_ALIASES):
+        recipients_by_email = braze_client_instance.create_recipients(
+            alias_label=ENTERPRISE_BRAZE_ALIAS_LABEL,
+            user_id_by_email=user_id_by_email_chunk
+        )
+        for _, recipient in recipients_by_email.items():
+            recipients.append(recipient)
```

Comment:
hmm, this is identical to the chunk above, maybe you can make a helper method and use it like so:
```python
def _recipients_for_identified_users(user_id_by_email):
    recipients = []
    for user_id_by_email_chunk in batch_dict(user_id_by_email, MAX_NUM_IDENTIFY_USERS_ALIASES):
        recipients_by_email = braze_client_instance.create_recipients(
            alias_label=ENTERPRISE_BRAZE_ALIAS_LABEL,
            user_id_by_email=user_id_by_email_chunk
        )
        recipients.extend(recipients_by_email.values())
```
...and then you can call it from within the tasks by doing
```python
recipients.extend(_recipients_for_identified_users(user_id_by_email))
```